charts/linkerd2/templates/web-rbac.yaml

```diff
@@ -101,8 +101,8 @@ subjects:
 - kind: ServiceAccount
   name: linkerd-web
   namespace: {{.Values.global.namespace}}
----
 {{- end}}
+---
```

Comment:
Hmm, looks like a unexpected change?

Reply:
Actually its a called for change as the end block was dangling down that manifest separator, i corrected it.